test/strategies.js

```diff
@@ -3956,6 +3956,42 @@ const createCurveUsdFLDebtBoostStrategy = () => {
     boostStrategy.addAction(curveUsdCollRatioCheckAction);
     return boostStrategy.encodeForDsProxyCall();
 };
+
+const createCurveUsdPaybackStrategy = () => {
+    const paybackStrategy = new dfs.Strategy('CurveUsdPaybackStrategy');
+
+    paybackStrategy.addSubSlot('&controllerAddr', 'address');
+    paybackStrategy.addSubSlot('&addressToPullTokensFrom', 'address');
+    paybackStrategy.addSubSlot('&amountToPayback', 'uint256');
+    paybackStrategy.addSubSlot('&crvUsdAddress', 'address');
+
+    const curveUsdHealthRatioTrigger = new dfs.triggers.CurveUsdHealthRatioTrigger(nullAddress, nullAddress, '0');
+    paybackStrategy.addTrigger(curveUsdHealthRatioTrigger);
+
+    const pullTokenAction = new dfs.actions.basic.PullTokenAction(
+        '&crvUsdAddress', // taken from subdata
+        '&addressToPullTokensFrom', // taken from subdata
+        '%amountToPayback', // sent by backend, either amount from sub data or maxUint for whole balance
+    );
+    const feeTakingAction = new dfs.actions.basic.GasFeeAction(
+        '0', // sent by backend
+        '&crvUsdAddress', // taken from subdata
+        '$1', // output of pull token action
+    );
+    const curveUsdPaybackAction = new dfs.actions.curveusd.CurveUsdPaybackAction(
+        '&controllerAddr', // taken from subdata,
+        '&proxy', // piped
+        '&proxy', // piped
+        '&addressToPullTokensFrom', // taken from subdata
```

Comment:
I think this should be &eoa, it will most likely never be used, but -> scenario in which a user using multisig subscribed to safe payback positions debt from his EOA, gets his position closed accidentaly by us, it makes sense for the coll/crvusd to go to multisig safe and not his EOA